content/course-specific-material/introR-LMGWSCAR-Oct16.Rmd

```diff
@@ -21,20 +21,19 @@ See [Before the Workshop](/intro-curriculum/Before) for information on what soft
 * 12:00 pm - 01:00 pm -- *Lunch*
 * 01:00 pm - 03:00 pm -- [Clean](/intro-curriculum/Clean)
 * 03:00 pm - 03:15 pm -- *Break*
-* 03:15 pm - 04:15 pm -- [Clean](/intro-curriculum/Clean) continued
+* 03:15 pm - 04:15 pm -- [Explore](/intro-curriculum/Explore)
```

Comment:
I would just caution you that Clean generally takes longer, and from the past 3 trainings we've agreed that the whole afternoon is best.